supertokens_python/recipe/session/utils.py

```diff
@@ -186,18 +203,30 @@ def __init__(
             ],
             None,
         ] = None,
+        on_clear_duplicate_session_cookies: Union[
+            None,
+            Callable[
+                [BaseRequest, str, BaseResponse],
+                Union[BaseResponse, Awaitable[BaseResponse]],
+            ],
+        ] = None,
     ):
         if on_token_theft_detected is None:
             on_token_theft_detected = default_token_theft_detected_callback
         if on_unauthorised is None:
             on_unauthorised = default_unauthorised_callback
         if on_invalid_claim is None:
             on_invalid_claim = default_invalid_claim_callback
+        if on_clear_duplicate_session_cookies is None:
```

Comment:
not related to python, but since node pr is merged, writing it here: This step is missing in node

Reply:
Fixed it in the node pr.